tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -1123,3 +1124,356 @@ def check_extracted_metadata(self, metadata: dict):
+#         )
+
+
+class TestFemtonicsImagingInterfaceStaticMethods:
+    """Test static methods of FemtonicsImagingInterface."""
+
+    def test_get_available_channels_p29(self):
+        """Test getting available channels for p29.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p29.mesc"
+        channels = FemtonicsImagingInterface.get_available_channels(file_path=file_path)
+        assert channels == ["UG", "UR"]
+
+    def test_get_available_channels_p30(self):
+        """Test getting available channels for p30.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p30.mesc"
+        channels = FemtonicsImagingInterface.get_available_channels(file_path=file_path)
+        assert channels == ["UG", "UR"]
+
+    def test_get_available_sessions_p29(self):
+        """Test getting available sessions for p29.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p29.mesc"
+        sessions = FemtonicsImagingInterface.get_available_sessions(file_path=file_path)
+        assert sessions == ["MSession_0"]
+
+    def test_get_available_sessions_p30(self):
+        """Test getting available sessions for p30.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p30.mesc"
+        sessions = FemtonicsImagingInterface.get_available_sessions(file_path=file_path)
+        assert sessions == ["MSession_0"]
+
+    def test_get_available_munits_p29(self):
+        """Test getting available units for p29.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p29.mesc"
+        units = FemtonicsImagingInterface.get_available_munits(file_path=file_path, session_name="MSession_0")
+        assert units == ["MUnit_0", "MUnit_1"]
+
+    def test_get_available_munits_p30(self):
+        """Test getting available units for p30.mesc."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p30.mesc"
+        units = FemtonicsImagingInterface.get_available_munits(file_path=file_path, session_name="MSession_0")
+        assert units == ["MUnit_0", "MUnit_1"]
+
+    def test_channel_name_not_specified_multiple_channels(self):
+        """Test that ValueError is raised when channel_name is not specified and multiple channels are available."""
+        file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p29.mesc"
+        with pytest.raises(
+            ValueError,
+            match=r"Multiple channels found in MSession_0/MUnit_0: \['UG', 'UR'\]\. Please specify 'channel_name' to select one\.",
+        ):
+            FemtonicsImagingInterface(
+                file_path=file_path,
+                munit_name="MUnit_0",
+            )
+
```

Comment:
We are missing tests for:
1) wrong channel name
2) Munit not specified.
3) Wrong MUnit name
4) Wrong Session.

Right? Am I missing something? what do you think?